src/test/datascience/notebook/nonPythonKernels.vscode.test.ts

```diff
@@ -173,7 +169,7 @@ suite('DataScience - VSCode Notebook - Kernels (non-python-kernel) (slow)', () =
         this.timeout(30_000); // Can be slow to start Julia kernel on CI.
         await openNotebook(api.serviceContainer, testJuliaNb.fsPath);
         await insertCodeCell('123456', { language: 'julia', index: 0 });
-        await waitForKernelToGetAutoSelected();
+        await waitForKernelToGetAutoSelected('julia');
```

Comment:
Found that C# was defaulting to Julia kernel when Jupyter wasn't available.
This way when tests fail, we know exactly why its failing. I.e. fail early when the wrong kernel is selected.